service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/RevurderingStatistikkMapper.kt

```diff
@@ -39,7 +40,17 @@ internal class RevurderingStatistikkMapper(private val clock: Clock) {
                             is IverksattRevurdering.Opphørt -> "Opphørt"
                             is IverksattRevurdering.IngenEndring -> "Uendret"
                         },
-                        resultatBegrunnelse = "Endring i søkers inntekt", // TODO ai: Må støtte flere grunner for revurdering senare
+                        resultatBegrunnelse = when (revurdering.revurderingsårsak.årsak) {
```

Comment:
Kan slenge på en kommentar om dette kun er kvalitative data, eller om de bruker det kvantitativt.

Reply:
@aryani1 Er dette egentlig helt riktig? Mulig at vi ikke har noe bedre data, men "begrunnelsen for resultatet" er vel egentlig ikke det samme som "årsaken for revurdering"?

Reply:
Hmm, nei du har nog rätt. Denne skal egentligen vara något i stil med: Hvorfor blev den innviliget? eller hvorfor avslag?

Reply:
Tänkte mer att når en revurdering blir innvilget f.eks, så er grunnen till innvilgelsen den faktiske årsaken. 😅  if that makes sense.

Reply:
Førslag på hva vi burde si da i `resultatBegrunnelse` @christianmosveen?

Reply:
Det er jo en kombinasjons av inngangsvilkårene og beregninga 🤷

Reply:
Sender vi avslagsgrunnen ved avslag på søknadsbehandlingen? Antar kanskje denne også vil være problematisk siden vi jo kan ha mange avslagsgrunner? 
Uansett; vi kan sjekke med de også her da, om dette feltet er så viktig. Hvis ja, så må vi vel lage noe som utleder det på et vis (opphør - opphørsgrunn, endring - hva ble endret, eller noe sånt).